mama/c_cpp/src/c/mama.c

```diff
@@ -235,6 +242,16 @@ mama_loadPayloadBridgeInternal  (mamaPayloadBridge* impl,
 mama_status
 mama_loadEntitlementBridgeInternal  (const char* name);
 
+void MAMACALLTYPE mamaImpl_entitlementDisconnectCallback (
```

Comment:
As discussed, will rename these to avoid callback function name conflicts just for the sake of clarity.